netci.groovy

```diff
@@ -2142,6 +2151,53 @@ combinedScenarios.each { scenario ->
                                         }
                                     }
                                 }
+                                else if (architecture == 'x86' && os == 'Ubuntu') {
```

Comment:
I believe there is enough redundancy with the code just above that we should factor out the differences and merge the blocks with appropriate `if`'s.  If you strongly disagree, let me know.

If you're worried about affecting working code, let's stage these changes in the dev/unix_test_workflow branch.  Once they work, we can port the change to master.

@jashook should be able to help with that.